packages/discord.js/src/structures/Sticker.js

```diff
@@ -182,8 +182,9 @@ class Sticker extends Base {
    * Fetches the pack that contains this sticker.
    * @returns {Promise<?StickerPack>} The sticker pack or `null` if this sticker does not belong to one.
    */
-  async fetchPack() {
-    return (this.packId && (await this.client.fetchStickerPacks()).get(this.packId)) ?? null;
+  fetchPack() {
```

Comment:
```suggestion
  async fetchPack() {
```

Reply:
This goes against the eslint configuration